models/site.js

```diff
@@ -16,7 +17,14 @@ module.exports = (sequelize, DataTypes) => {
       Site.belongsToMany(models.Service, { through: models.SiteService });
       Site.hasMany(models.Hours);
     }
+
+    toJSON() {
+      const json = _.pick(this.get(), ['id', 'name', 'address', 'phoneNumber', 'email', 'website']);
+      json.NutritionPartnerIds = this.NutritionPartners?.map((np) => np.id) ?? [];
+      return json;
+    }
```

Comment:
On the backend, I add a custom toJSON() function to the Site model so that it can properly return the NutritionParnerIds with the rest of the attributes.

Reply:
I've decided to simplify the implementation so I removed this custom toJSON function.